src/SignalR/server/Core/src/StreamTracker.cs

```diff
@@ -16,14 +16,20 @@ namespace Microsoft.AspNetCore.SignalR
     internal class StreamTracker
     {
         private static readonly MethodInfo _buildConverterMethod = typeof(StreamTracker).GetMethods(BindingFlags.NonPublic | BindingFlags.Static).Single(m => m.Name.Equals("BuildStream"));
+        private readonly int _streamBufferCapacity = 10;
         private ConcurrentDictionary<string, IStreamConverter> _lookup = new ConcurrentDictionary<string, IStreamConverter>();
 
+        public StreamTracker(int streamBufferCapacity)
+        {
+            _streamBufferCapacity = streamBufferCapacity;
+        }
+
         /// <summary>
         /// Creates a new stream and returns the ChannelReader for it as an object.
         /// </summary>
         public object AddStream(string streamId, Type itemType)
         {
-            var newConverter = (IStreamConverter)_buildConverterMethod.MakeGenericMethod(itemType).Invoke(null, Array.Empty<object>());
+            var newConverter = (IStreamConverter)_buildConverterMethod.MakeGenericMethod(itemType).Invoke(null, new object[] { _streamBufferCapacity });
```

Comment:
Nit: can we reuse the array here rather than newing up one every time? Maybe create it in the constructor?